pkg/arvo/sys/vane/clay.hoon

```diff
@@ -1962,32 +1968,163 @@
     =/  =wire  /merge/[syd]/(scot %p ali-ship)/[ali-desk]/[germ]
     (emit hen %pass wire %c %warp ali-ship ali-desk `[%sing %v case /])
   ::
+  ++  make-melt
+    |=  [bas=beak con=(list [beak germ])]
+    ^-  melt
+    :+  bas  con
+    %-  ~(gas by (~(put by *(map beak (unit dome:clay))) bas *(unit dome:clay)))
+    (turn con |=(a=[beak germ] [-.a *(unit dome:clay)]))
+  ::
+  ++  start-fuse
+    |=  [bas=beak con=(list [beak germ])]
+    ^+  ..start-fuse
+    =/  moves=(list move)
+        %+  turn
+          [[bas *germ] con]
+        |=  [bec=beak germ]
+        ^-  move
+        =/  wir=wire  /fuse/[syd]/(scot %p p.bec)/[q.bec]/(scot r.bec)
+        [hen %pass wir %c %warp p.bec q.bec `[%sing %v r.bec /]]
+    ::  we also want to clear the state (fiz) associated with this
+    ::  merge and print a warning if it's non trivial i.e. we're
+    ::  starting a new fuse before the previous one terminated.
+    ~&  ?~  con.fiz
+          [%starting-fuse bas con]
+        :*  %starting-fuse
+            bas
+            con
+            %discarding-state
+            ::  we don't want to ~& an entire dome
+            %-  ~(run by sto.fiz)
+            |=(v=(unit dome:clay) ?~(v %not-received %recieved))
+        ==
+    =.  fiz  (make-melt bas con)
+    (emil moves)
+  ::
+  ++  take-fuse
+    |=  [bec=beak =riot]
+    ^+  ..take-fuse
+    ?~  riot
+      ~&  [%fuse-for syd %missing bec]
+      ::  by setting fiz to *melt the merge is aborted - any further
+      ::  responses we get for the merge will cause take-fuse to crash
+      =.  fiz  *melt
+      ..take-fuse
+    ?>  (~(has by sto.fiz) bec)
+    =.  fiz
+        :+  bas.fiz  con.fiz
+        (~(put by sto.fiz) bec `!<(dome:clay q.r.u.riot))
+    =/  all-done=flag
+        %-  ~(all by sto.fiz)
+        |=  res=(unit dome:clay)
+        ^-  flag
+        !=(res ~)
```

Comment:
yeah I'll keep my bastardized style to ucal exclusively :P 

fwiw the runes that "introduce new variables" just make more sense to me if the name and the value have different indentation levels because I can more easily see what the variable is instead of trying to read the entire sub expression. not trying to advocate for this to be generally used but would be interested to hear if other people have the same opinion. perhaps a question for the forge..

Reply:
I'd say consistency is the most important thing for readability, no matter the rune.

Reply:
Perhaps should be more clear.  This is the correct style:

```
=/  new-name=type
  ?~  some
    long
  value
continuation
```

Some of the changes you made dedented too far, to

```
=/  new-name=type
?~  some
  long
value
continuation
```

Which I agree is hard to read because it mixes up the value with the remaining code.  In the principle above, I count the remaining code as the (first-to-) last child and the value of the variable as the 2nd-to-last child, which is why it's indented one level, but not two.